### PR TITLE
First minimal version adding response examples to Postman collections.

### PR DESCRIPTION
**Pull request by jlengrand**
* ~~Need to add test~~
*  ~~Need to handle codes properly~~
* ~~Need to ensure responses are placed in the right location~~

### CODE REVIEW

File: src/main/resources/postman-v2/item.mustache [hunk 1]
```diff
@@ -5,6 +5,13 @@
                         {{#vendorExtensions.postmanRequests}}
                             {
                                 "name": "{{{name}}}",
+                                "response": [{
+                                    "name": "{{response.name}}",
+                                    "code": "{{response.code}}",
```
Comment (gcatanese) [inline src/main/resources/postman-v2/item.mustache]: @jlengrand code should be `integer` (Postman JSON schema v2.1.0)

File: src/main/resources/postman-v2/item.mustache [hunk 2]
```diff
@@ -5,60 +5,24 @@
...
-                                    "url": {
-                                        "raw": "{{=<% %>=}}{{baseUrl}}<%={{ }}=%>{{path}}",
-                                        "host": [
-                                            "{{=<% %>=}}{{baseUrl}}<%={{ }}=%>"
-                                        ],
-                                        "path": [
-                                            {{#vendorExtensions.pathSegments}}
-                                            "{{.}}"{{^-last}},{{/-last}}
-                                            {{/vendorExtensions.pathSegments}}
+                                "request": {{>request}}
```
Comment (gcatanese) [inline src/main/resources/postman-v2/item.mustache:8]: awesome refactoring here

File: src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+     */
+    public static String getStatus(CodegenResponse codegenResponse) {
+        String ret = "";
+
+        if (codegenResponse.is2xx) {
+            if (codegenResponse.code.equalsIgnoreCase("200")) {
+                ret = "OK";
+            } else if (codegenResponse.code.equalsIgnoreCase("201")) {
+                ret = "Created";
+            } else {
```
Comment (gcatanese) [inline src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java:21]: @jlengrand should also deal with `202` Accepted (used by few Adyen APIs)

**Comment by gcatanese** [comment]
Awesome work, I left some comments to consider/address

File: src/main/resources/postman-v2/item.mustache (hunk 2 above)
Comment (jlengrand) [reply src/main/resources/postman-v2/item.mustache:8]: thanks. The alternative was duplication and I hate it

File: src/main/resources/postman-v2/item.mustache (hunk 1 above)
Comment (jlengrand) [reply src/main/resources/postman-v2/item.mustache]: Good point!

File: src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package com.tweesky.cloudtools.codegen;
```
Comment (jlengrand) [inline src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java:1]: @gcatanese I moved to something more generic, which should be better. 

However, it does modify the behaviour somehow (for example "success" default cases).

can you confirm whether that is desired?

Comment (gcatanese) [reply src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java:1]: Looks good, an unexpected status code thrown an exception, which is probably better.

**Review by gcatanese — APPROVED** [review]

File: src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java (hunk 4 above)
Comment (jlengrand) [reply src/main/java/com/tweesky/cloudtools/codegen/CodegenUtils.java:1]: ah, forgot that case, I'll an extra test!

**Comment by jlengrand** [comment]
Hey @gcatanese before merging this do I need to do anything more? Create the new docker layer or something?

**Comment by gcatanese** [comment]
> Hey @gcatanese before merging this do I need to do anything more? Create the new docker layer or something?

Upon merge on `main` a new Docker image is built and pushed (Github action)

**Pull request merged**